README.asciidoc

```diff
@@ -123,7 +123,7 @@ to a single XML file for DocBook to process.  _Chunking_ is the process of
 breaking that XML file up into multiple HTML pages.
 
 By default, each _part_ (`= Part Title`) and _chapter_ (`== Chapter Title`) is
-``chunked'' into a separate HTML file. However, for bigger books, such as the
+"chunked" into a separate HTML file. However, for bigger books, such as the
```

Comment:
The ``word'' syntax is not supported by Asciidoctor. It inserted a smart double quote. The `"` just renders a dumb one which seems fine.